eng/common/scripts/Helpers/ApiView-Helpers.ps1

```diff
@@ -117,3 +117,106 @@ function Process-ReviewStatusCode($statusCode, $packageName, $apiApprovalStatus,
   $packageNameStatus.IsApproved = $packageNameApproved
   $packageNameStatus.Details = $packageNameApprovalDetails
 }
+
+function Set-ApiViewCommentForRelatedIssues {
+  param (
+    [Parameter(Mandatory = $true)]
+    [string]$HeadCommitish,
+    [string]$APIViewHost = "https://apiview.dev",
+    [ValidateNotNullOrEmpty()]
+    [Parameter(Mandatory = $true)]
+    $AuthToken
+  )
+  . ${PSScriptRoot}\..\common.ps1
+  $issuesForCommit = $null
+  try {
+    $issuesForCommit = Search-GitHubIssues -CommitHash $HeadCommitish
```

Comment:
By issues are we talking about pull requests?

Reply:
I guess the github api is called issue search and it take a commit. I know they consider PRs issues but I don't think a commit can be associated with a standard issue.

Reply:
The endpoint is `search/issues` that's why I used issues